sacred/messagequeue.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# coding=utf-8
+import sys
+
+if sys.version_info[0] == 2:
+    from Queue import Queue
+else:
+    from queue import Queue, Empty
+
+
+class SacredMQ:
+    """A simple message queue for Sacred.
+
+    The queue allows publishing messages
+    to multiple consumers. Each of the
+    consumers is responsible for checking
+    the queue. Otherwise, it might grow forever.
```

Comment:
They are far from 79. Honestly, I don't remember why I was so over-careful with the line width here.